src/process/executable_path.cr

```diff
@@ -31,20 +31,28 @@ class Process
   # Searches an executable, checking for an absolute path, a path relative to
   # *pwd* or absolute path, then eventually searching in directories declared
   # in *path*.
-  def self.find_executable(name, path = ENV["PATH"]?, pwd = Dir.current)
-    if name.starts_with?(File::SEPARATOR)
-      return name
+  def self.find_executable(name : Path | String, path : String = ENV["PATH"]?, pwd : Path | String = Dir.current) : String?
```

Comment:
```suggestion
  def self.find_executable(name : Path | String, path : Path | String? = ENV["PATH"]?, pwd : Path | String = Dir.current) : String?
```

Reply:
Nilable is correct. But I don't think it should accept `Path` because the value is a path list, which can't be expressed as that data type. So I'd rather not give that option.
Maybe we could redesign this interface even more and make it `Enumerable(Path)` or so but I'd leave that for another day.